### ISSUE TITLE
Demo script to classify one image

### ISSUE THREAD

**Issue by kadirbeytorun** [SoftwareGift/FeatherNets_Face-Anti-spoofing-Attack-Detection-Challenge-CVPR2019#55]
Hello, could you provide a simple script to classify a single image?

Really confused about the 1024 vector feature map, and all other issues are in chinese, so I dont know if anyone else asked about this.

Thanks in advance.

**Comment by SoftwareGift** [comment]
If you are only predicting a single image, you only need to take the first two categories of the category.

**Comment by kadirbeytorun** [comment]
Okay, but do what with them, give only two of them to softmax? Are they like preds[0]=fake, and preds[1]=real?

Also, do I need to detect face in the image first and then give the cropped face to network?

**Comment by SoftwareGift** [comment]
> Okay, but do what with them, give only two of them to softmax? Are they like preds[0]=fake, and preds[1]=real?
> 
> Also, do I need to detect face in the image first and then give the cropped face to network?

yes, you are right.

**Comment by kadirbeytorun** [comment]
Results are wrong. Could you please explain more if its okay?
Should I give normal rgb cropped face?
Or do I need to use depth-rgb cropped faces?
Regards

**Comment by jeffryuiop** [comment]
Hi, thank you for posting this.
However, if i want to make an inference demo script for one image using your pretrained weight, How do I do that? After passing the two elements through a softmax, how do I interpret the result? Currently I have tried to use a real and fake cropped image and the result is more or less the same.

Once again, thank you and best regards.

**Comment by SoftwareGift** [comment]
https://github.com/SoftwareGift/FeatherNets_Face-Anti-spoofing-Attack-Detection-Challenge-CVPR2019/issues/72#issuecomment-548178960
